changes/924.feature.md

```diff
@@ -1 +1 @@
-Add new command types and application autocomplete option
+Add new command types and application autocomplete option.
```

Comment:
wouldn't something like "Add context menu commands and command autocomplete." be better? Who even wrote this?